pkg/persistence/dgraph/pod.go

```diff
@@ -68,9 +75,13 @@ func CreatePod(pod api_v1.Pod, xid string) (*api.Assigned, error) {
 	return MutateNode(Client, bytes)
 }
 
+// PersistPod updates the pod details and create it a new node if not exists.
 func PersistPod(pod api_v1.Pod) error {
 	xid := pod.Namespace + ":" + pod.Name
-	uid, _ := GetUId(Client, xid, IsPod)
+	uid, err := GetUID(Client, xid, IsPod)
+	if err != nil {
```

Comment:
no need for err check, GetUID returns uid as "" if err is not nil

Reply:
Then we should not be returning the `error` from the `GetUID` method as the linter gives an error since we do not handle the error msg. Will fix the method and reverse this change.